src/app/client/src/app/modules/core/components/prominent-filter/prominent-filter.component.ts

```diff
@@ -141,14 +141,14 @@ export class ProminentFilterComponent implements OnInit, OnDestroy {
       this.prominentFilter.emit(this.formFieldProperties);
```

Comment:
add this line
  @Input() frameworkName: string;

add below line in ngOninit
    if (this.frameworkName) {
      this.frameworkService.initialize(this.frameworkName);
    } else  {
      this.frameworkService.initialize();
    }